src/agents/plots_videos.jl

```diff
@@ -73,7 +79,11 @@ evolving the ABM and a heatmap in parallel with only a few lines of code.
 """
 function abm_plot(model; resolution = (600, 600), kwargs...)
     fig = Figure(; resolution)
-    ax = fig[1,1][1,1] = Axis(fig)
+    ax = fig[1,1][1,1] = if dimensionality(model) == 3
+        Axis3(fig)
+    else
+        Axis(fig)
+    end
```

Comment:
```suggestion
    ax = fig[1,1][1,1] = dimensionality(model) == 3 ? Axis3(fig) : Axis(fig)
```

Reply:
Please use this in all the `if Axis/Axis3`.

Reply:
I left the one in `abm_video` since the function calls there are significantly longer and I can't seem to be able to format it in a way that is readable with the conditional/ternary operator

Reply:
I left the one in `abm_video` since the function calls there are significantly longer and I can't seem to be able to format it in a way that is readable with the conditional/ternary operator